dask_sql/physical/rel/custom/show_schemas.py

```diff
@@ -16,23 +16,29 @@ class ShowSchemasPlugin(BaseRelPlugin):
     Show all schemas.
     The SQL is:
 
-        SHOW SCHEMAS
+        SHOW SCHEMAS [FROM <catalog-name>] [LIKE <>]
 
     The result is also a table, although it is created on the fly.
     """
 
     class_name = "ShowSchemas"
 
     def convert(self, rel: "LogicalPlan", context: "dask_sql.Context") -> DataContainer:
-
         show_schemas = rel.show_schemas()
 
         # "information_schema" is a schema which is found in every presto database
         schemas = list(context.schema.keys())
         schemas.append("information_schema")
         df = pd.DataFrame({"Schema": schemas})
 
-        # We currently do not use the passed additional parameter FROM.
+        # currently catalogs other than the default `dask_sql` are not supported
+        catalog_name = show_schemas.getFrom() or context.catalog_name
+        if catalog_name != context.catalog_name:
+            raise RuntimeError(
+                f"A catalog with the name {catalog_name} is not present."
+            )
```

Comment:
For now, we are able to parse statements like `SHOW SCHEMAS FROM "dask_sql"`, but fail in any case where the catalog name isn't `dask_sql`; I figured it would make more sense to actually bother refactoring the Python handling of catalogs once we add functionality to add/remove/switch catalogs through queries.